sdk/communication/Azure.Communication.PhoneNumbers/phone-numbers-livetest-matrix.json

```diff
@@ -0,0 +1,77 @@
+{
```

Comment:
@weshaggard FYI. This is basically a copy of the now-verbose base NET matrix config, layered with alternating test phone numbers per OS. I still don't have a good solution here, other than adding some sort of "layer sparse on sparse".

Reply:
OK. We should definitely keep thinking about this because the more we repeat these entries the harder it is going to be to maintain the default set of configurations.